src/gpu/intel/conv/jit/plan.cpp

```diff
@@ -154,21 +154,21 @@ void init_fwd(const config_t &cfg_, gemm_schedule_t &gemm_schedule,
     auto &dst_layout = cfg_.dst_layout().compute();
 
     // Initialize views.
-    auto mb = var_t::make(type_t::s32(), "mb");
-    auto ic = var_t::make(type_t::s32(), "ic");
-    auto oc = var_t::make(type_t::s32(), "oc");
-    auto kd = var_t::make(type_t::s32(), "kd");
-    auto kh = var_t::make(type_t::s32(), "kh");
-    auto kw = var_t::make(type_t::s32(), "kw");
-    auto g = var_t::make(type_t::s32(), "g");
+    auto mb = var_t::make(dsl::type_t::s32(), "mb");
```

Comment:
As an option we can introduce aliases for common DSL types inside gpu::intel (such as type_t or layout_t, etc) to reduce boilerplate.

Reply:
My current plan is to generally add:
```
namespace dsl = gemmstone::dsl;
namespace ir = gemmstone::dsl::ir;
```
and leave further simplification up to specific implementations. I am not convinced we should broadly be overriding these as we already have some duplicate naming (especially in the conv::v2 model code) and to generally avoid inconsistency in places where there is a naming clash on specific names.